content/post/graceful-k8s-delpoyments/index.md

```diff
@@ -0,0 +1,205 @@
+---
+title: "Zero-Downtime Rolling Deployments in Kubernetes"
+date: 2021-07-1500:00:00+09:00
+hero: /images/rolling-waves.jpg
```

Comment:
Why is the hero image not in `content/post/graceful-k8s-delpoyments/` and referenced with a relative link ?

Reply:
because it's also used other places so it needs an absolute link (e.g. on the home page and I think at the bottom of a post in the "more articles" section). I can move the picture inside the posts folder, but will need an absolute path regardless

Reply:
Hugo limitation, it can't resolve files —got it!